src/proxy.sol

```diff
@@ -15,7 +15,7 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-pragma solidity >=0.5.0 <0.6.0;
+pragma solidity >=0.6.0;
```

Comment:
you should set a higher bound, since the contract doesn't compile with solidity 0.8 for example (due to the cast in line 113)